### PR TITLE
renderTabName ScrollableTabBar property

### LINKED ISSUE #240: tab property that accepts a render componennt
It's nice that you render a custom tab bar at the top level, but it would make a lot of sense to have the option to customize rendering at the tab level instead. It would require a lot less customization for the developer and seems more intuitive

### PR DESCRIPTION
**Pull request by anyx**
closes #240

### CODE REVIEW

**Comment by skv-headless** [comment]
Hi. What for is it?

**Comment by anyx** [comment]
@skv-headless hi. For rendering custom tab labels.
I want to show tab title with badge, for example.

Is there other ways to do that?

**Comment by skv-headless** [comment]
Now I see. Cool 👍 
Could you make this property for DefaultTabBar?
Please describe it in propTypes and defaultProps

**Comment by anyx** [comment]
I updated pr

File: DefaultTabBar.js [hunk 1]
```diff
@@ -29,31 +30,47 @@ const DefaultTabBar = React.createClass({
...
 
   renderTabOption(name, page) {
     const isTabActive = this.props.activeTab === page;
-    const { activeTextColor, inactiveTextColor, textStyle, } = this.props;
-    const textColor = isTabActive ? activeTextColor : inactiveTextColor;
-    const fontWeight = isTabActive ? 'bold' : 'normal';
 
     return <Button
-      style={{flex: 1, }}
+      style={{flex: 1}}
```
Comment (skv-headless) [inline DefaultTabBar.js]: there is an eslint and such notation doesn't conform it

File: DefaultTabBar.js [hunk 2]
```diff
@@ -29,31 +30,47 @@ const DefaultTabBar = React.createClass({
       underlineColor: 'navy',
       backgroundColor: null,
       underlineHeight: 4,
+      renderTabName: null
```
Comment (skv-headless) [inline DefaultTabBar.js]: ```renderTabName: this.renderTabName``` would be cooler and ```renderTabName``` will become shorter

**Comment by anyx** [comment]
updated

File: ScrollableTabBar.js [hunk 3]
```diff
@@ -136,14 +135,24 @@ const ScrollableTabBar = React.createClass({
...
+      {this.renderTabName(name, page, isTabActive)}
     </Button>;
   },
 
+  renderTabName(name, page, isTabActive) {
+    const { activeTextColor, inactiveTextColor, textStyle, } = this.props;
+    const textColor = isTabActive ? activeTextColor : inactiveTextColor;
+    const fontWeight = isTabActive ? 'bold' : 'normal';
+
+    return (
```
Comment (skv-headless) [inline ScrollableTabBar.js]: last nitpick. I'm trying to follow khan academy style guide
https://github.com/Khan/style-guides/blob/master/style/react.md#open-elements-on-the-same-line
Could you please fix return statement

**Comment by anyx** [comment]
fixed

**Comment by skv-headless** [comment]
Thanks 👍 
It would be awesome if you make a simple example how to do it here https://github.com/skv-headless/react-native-scrollable-tab-view/blob/master/examples/FacebookTabsExample/DynamicExample.js

**Comment by antoinerousseau** [comment]
would be nice to update the docs. and a changelog...

**Pull request merged**